doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,306 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
 The dashboard provides a visual representation of the system state, allowing users to track the performance 
 of their applications and troubleshoot issues.
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/Dashboard-overview.png
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/overview_gif.gif
```

Comment:
@rkooo567 The gif is a bit blurry. Can we get a higher-res one? I usually use tools like gif brewery 3

Reply:
Let me try again... Btw I think we should take some youtube videos eventually and use it instead of gif

Reply:
Hmm looks like gif brewery 3 is not available in Japan. Is there other apps you recommend?

Reply:
Video sent in for you to take a look :) Let me know what edits you'd like to see or format you prefer.